test/views/changeActivity/ThirdPartyPackagersViewSpec.scala

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.changeActivity
+
+import controllers.changeActivity.routes
+import forms.changeActivity.ThirdPartyPackagersFormProvider
+import models.{CheckMode, NormalMode}
+import play.api.data.Form
+import play.api.i18n.Messages
+import play.api.mvc.Request
+import play.api.test.FakeRequest
+import views.ViewSpecHelper
+import views.html.changeActivity.ThirdPartyPackagersView
+
+
+class ThirdPartyPackagersViewSpec extends ViewSpecHelper {
+
+  val view: ThirdPartyPackagersView =   application.injector.instanceOf[ThirdPartyPackagersView]
+  val formProvider = new ThirdPartyPackagersFormProvider
+  val form = formProvider.apply()
+  implicit val request: Request[_] = FakeRequest()
+
+  object Selectors {
+    val heading = "govuk-fieldset__heading"
+    val legend = "govuk-fieldset__legend  govuk-fieldset__legend--m"
+    val radios = "govuk-radios__item"
+    val radioInput = "govuk-radios__input"
+    val radioLabels = "govuk-label govuk-radios__label"
+    val body = "govuk-body"
+    val errorSummaryTitle = "govuk-error-summary__title"
+    val errorSummaryList = "govuk-list govuk-error-summary__list"
+    val button = "govuk-button"
+    val form = "form"
+  }
+
+  "View" - {
+    val html = view(form, NormalMode)(request, messages(application))
+    val document = doc(html)
+    "should contain the expected title" in {
+      document.title() must include(Messages("changeActivity.thirdPartyPackagers" + ".title"))
```

Comment:
This calls the messages file instead of it checking what we actually anticipate.  Please write out the expected title